website/layouts/index.html

```diff
@@ -49,6 +49,11 @@ <h3 class="mb-3">Open source, highly available Prometheus setup with long term s
     or
     <a class="color-yellow" target="_blank" href="https://care-in-action.org/en/pages/please-fund-emergency-aid-for-ukraine">Care in Action</a> to provide relief.
 </div>
+<div class="bg-yellow text-dark py-3 font-weight-bold container-fluid text-center">
```

Comment:
From deploy preview, this looks to be same color as rest of the page. I wonder if we can do `bg-info` instead (would be cyan)?
![Screenshot 2023-11-21 at 3 42 37 PM](https://github.com/thanos-io/thanos/assets/51132453/7bcf3831-36a0-451b-9f03-81f0a1000f38)

```suggestion
<div class="bg-info text-white py-3 font-weight-bold container-fluid text-center">
```

Reply:
That would do, thanks!